internal/cmd/server.go

```diff
@@ -74,18 +81,19 @@ func newServerCmd() *cobra.Command {
 	cmd.Flags().String("db-encryption-key-provider", "", "Database encryption key provider")
 	cmd.Flags().Bool("enable-telemetry", false, "Enable telemetry")
 	cmd.Flags().Bool("ui-enabled", false, "Enable Infra server UI")
-	cmd.Flags().Var(&options.UI.ProxyURL, "ui-proxy-url", "Proxy upstream UI requests to this url")
+	cmd.Flags().Var(&types.URL{}, "ui-proxy-url", "Proxy upstream UI requests to this url")
 	cmd.Flags().Duration("session-duration", 0, "User session duration")
 	cmd.Flags().Bool("enable-signup", false, "Enable one-time admin signup")
 
 	return cmd
 }
 
-func defaultServerOptions() server.Options {
+func defaultServerOptions(infraDir string) server.Options {
 	return server.Options{
-		TLSCache:                "$HOME/.infra/cache",
-		DBFile:                  "$HOME/.infra/sqlite3.db",
-		DBEncryptionKey:         "$HOME/.infra/sqlite3.db.key",
+		Version:                 0.2, // update this as the config version changes
```

Comment:
Previously this was being set in the test function, so I think in the last release we weren't setting the version. I guess that's ok because we haven't modified any keys, it's a separate key in the config.